pages/in/oauth/_platform/_id/index.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <div class="lc-margin-vertical-32">
+    <div class="lc-container-0">
+      <div class="lc-container-1">
+        <div class="lc-container-2">
+          <div class="lc-container-3">
+            <div class="lc-container-4">
+              <span v-if="errorMsg">
+                {{ $t('General.label.error') }}: {{ errorMsg }},
+                <nuxt-link :to="{ name: 'index' }">
+                  {{ $t('Verify.label.toIndex') }}
+                </nuxt-link>
+                ...
+              </span>
+
+              <span v-else-if="isDone">
+                {{ $t('General.label.success') }},
+                <nuxt-link :to="{ name: 'in' }">
+                  {{ $t('Verify.label.toEdit') }}
+                </nuxt-link>
+                ...
+              </span>
+              <span v-else>{{ $t('Verify.label.verifying') }}</span>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
```

Comment:
Is this multi-level nesting and `...` intended and temporary?

Reply:
@nwingt i copied this from `/verify`, not sure if needed for layout